index.bs

```diff
@@ -183,7 +183,7 @@ The syntax and semantics of the <dfn export>AV1ItemConfigurationProperty</dfn> a
     - <assert>[=Sequence Header OBUs=] should not be present in the [=AV1CodecConfigurationBox=].</assert>
     - <assert>If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].</assert>
     - <assert>The values of the fields in the [=AV1CodecConfigurationBox=] shall match those of the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].</assert>
-    - <assert>The values of the bit depth information in the [=AV1CodecConfigurationBox=] shall match the PixelInformationProperty if present.</assert>
+    - <assert>The values of the bit depth information and the number of channels in the [=AV1CodecConfigurationBox=] shall match the PixelInformationProperty if present.</assert>
```

Comment:
Nit/optional: Omit the word "information", unless you wanted to stress the fact that the bit depth is calculated from multiple syntax elements in the AV1CodecConfigurationBox and use "bit depth information" to refer to those syntax elements collectively.

Reply:
Yeah, I was kind of hinting towards the bit depth requiring multiple fields, but not sure how useful that is. Shorter text is better when possible, so I've removed it.

Reply:
Thanks for confirming the intent. One way to clarify that is to change `in` to `derived from` before `the [=AV1CodecConfigurationBox=]`.

Reply:
Good point! Take a look at the final wording that I merged and see if you would prefer that. Let's change it if so.

Reply:
Yes, I prefer `derived from`. If you also prefer `derived from`, let's switch to that. It is more accurate because the bit depth is derived from three fields (`seq_profile`, `high_bitdepth`, and `twelve_bit`) and the number of channels is derived from the `monochrome` field.

Reply:
Filed new issue: #242